validator/src/main/java/com/google/daq/mqtt/registrar/Registrar.java

```diff
@@ -705,6 +725,9 @@ private void loadSiteMetadata() {
 
     File siteMetadataFile = new File(siteDir, SITE_METADATA_JSON);
     try (InputStream targetStream = new FileInputStream(siteMetadataFile)) {
+      // At this time, do not validate the Metadata schema because, by its nature of being
```

Comment:
I'm confused by this comment -- it says "do not validate" and then the code right after is validate()... ?

Reply:
Yes but we ignore the return value, the report.

Reply:
Still confused -- if it's ignored then why is it calling validate()?    Do you mean something like "At this time, only validate the structure of the Metadata schema because it's only a partial overlay, and defer any in-depth validation until after merge?"